frontend/src/metabase/visualizations/echarts/cartesian/option/series.ts

```diff
@@ -222,7 +222,10 @@ const buildEChartsLineAreaSeries = (
         show: settings["graph.show_values"] && !hasMultipleSeries,
       },
       itemStyle: {
-        opacity: isSymbolVisible ? 1 : 0,
+        opacity: isSymbolVisible ? 0.3 : 0,
```

Comment:
This is to fix the dots from being full opacity even in blurred state

Reply:
@EmmadUsmani when there is 1 series, we should not change opacity on symbols according to the existing implementation, so it seems the opacity should be:

```
const blurSymbolOpacity = hasMultipleSeries ? 0.3 : 1;
```

```
opacity: isSymbolVisible ? blurSymbolOpacity : 0
```

Reply:
or better reuse it for the series itself too
<img width="459" alt="Screenshot 2024-02-08 at 7 31 30 PM" src="https://github.com/metabase/metabase/assets/14301985/0f322efb-4944-4b2e-8e73-ee8a8d26987a">

Reply:
Thanks, I just fixed this